# Removes in order:
# 'Tabs from other devices' entry on the History page sidebar
# link to Google's accessibility site from the settings page
# link to Google's help site on the About page
# Safety check section on the settings page
# option to offer page translations on the settings page
# cloud printing entry on the settings page
# option for password leak detection on the settings page
# Google sign-in and Anonymized Data Collection sections
# Advanced Protection Program link on the security settings page
# Safety Check entry on the side menu on the settings page
# unneeded elements from the profile menu

--- a/chrome/browser/resources/history/side_bar.html
+++ b/chrome/browser/resources/history/side_bar.html
@@ -104,11 +104,6 @@
           $i18n{historyMenuItem}
           <paper-ripple></paper-ripple>
         </a>
-        <a role="menuitem" href="/syncedTabs" class="page-item"
-            path="syncedTabs" on-click="onItemClick_">
-          $i18n{openTabsMenuItem}
-          <paper-ripple></paper-ripple>
-        </a>
         <div class="separator"></div>
         <a role="menuitem" id="clear-browsing-data"
             href="chrome://settings/clearBrowserData"
--- a/chrome/browser/resources/settings/a11y_page/a11y_page.html
+++ b/chrome/browser/resources/settings/a11y_page/a11y_page.html
@@ -51,10 +51,6 @@
             sub-label="$i18n{accessibleImageLabelsSubtitle}">
         </settings-toggle-button>
 </if>
-        <cr-link-row class="hr" label="$i18n{moreFeaturesLink}"
-            on-click="onMoreFeaturesLinkClick_" sub-label="$i18n{a11yWebStore}"
-            external>
-        </cr-link-row>
       </div>
 <if expr="not chromeos">
       <template is="dom-if" if="[[!captionSettingsOpensExternally_]]">
--- a/chrome/browser/resources/settings/about_page/about_page.html
+++ b/chrome/browser/resources/settings/about_page/about_page.html
@@ -122,8 +122,6 @@
         </div>
       </template>
 </if>
-      <cr-link-row class="hr" id="help" on-click="onHelpTap_"
-          label="$i18n{aboutGetHelpUsingChrome}" external></cr-link-row>
 <if expr="_google_chrome">
       <cr-link-row class="hr" id="reportIssue" on-click="onReportIssueTap_"
           hidden="[[!prefs.feedback_allowed.value]]"
--- a/chrome/browser/resources/settings/basic_page/basic_page.html
+++ b/chrome/browser/resources/settings/basic_page/basic_page.html
@@ -95,6 +95,7 @@
             <settings-autofill-page prefs="{{prefs}}"></settings-autofill-page>
           </settings-section>
         </template>
+<if expr="false">
         <template is="dom-if" if="[[showPage_(pageVisibility.safetyCheck)]]"
             restamp>
           <settings-section page-title="$i18n{safetyCheckSectionTitle}"
@@ -104,6 +105,7 @@
             </settings-safety-check-page>
           </settings-section>
         </template>
+</if>
         <template is="dom-if" if="[[showPage_(pageVisibility.privacy)]]"
             restamp>
           <settings-section page-title="$i18n{privacyPageTitle}"
--- a/chrome/browser/resources/settings/privacy_page/personalization_options.html
+++ b/chrome/browser/resources/settings/privacy_page/personalization_options.html
@@ -8,7 +8,7 @@
         border-top: none;
       }
     </style>
-<if expr="not chromeos">
+<if expr="false">
     <settings-toggle-button id="signinAllowedToggle"
         class="hr"
         hidden="[[!signinAvailable_]]"
@@ -51,12 +51,6 @@
     </settings-toggle-button>
 </if><!-- not chromeos -->
 </if><!-- _google_chrome -->
-    <settings-toggle-button id="urlCollectionToggle"
-        class="hr"
-        pref="{{prefs.url_keyed_anonymized_data_collection.enabled}}"
-        label="$i18n{urlKeyedAnonymizedDataCollection}"
-        sub-label="$i18n{urlKeyedAnonymizedDataCollectionDesc}">
-    </settings-toggle-button>
 <if expr="_google_chrome">
     <settings-toggle-button id="spellCheckControl"
         class="hr"
--- a/chrome/browser/resources/settings/privacy_page/security_page.html
+++ b/chrome/browser/resources/settings/privacy_page/security_page.html
@@ -76,10 +76,3 @@
         sub-label="$i18n{manageCertificatesDescription}"
         on-click="onManageCertificatesClick_"></cr-link-row>
 </if>
-    <cr-link-row id="advanced-protection-program-link"
-        class="hr"
-        label="$i18n{advancedProtectionProgramTitle}"
-        sub-label="$i18n{advancedProtectionProgramDesc}"
-        on-click="onAdvancedProtectionProgramLinkClick_"
-        external>
-    </cr-link-row>
--- a/chrome/browser/resources/settings/settings_menu/settings_menu.html
+++ b/chrome/browser/resources/settings/settings_menu/settings_menu.html
@@ -111,14 +111,6 @@
             <iron-icon icon="settings:assignment"></iron-icon>
             $i18n{autofillPageTitle}
           </a>
-          <template is="dom-if" if="[[!enableLandingPageRedesign_]]">
-            <a role="menuitem" href="/safetyCheck"
-                hidden="[[!pageVisibility.safetyCheck]]"
-                id="safetyCheck">
-              <iron-icon icon="settings20:safety-check"></iron-icon>
-              $i18n{safetyCheckSectionTitle}
-            </a>
-          </template>
           <a role="menuitem" href="/privacy"
               hidden="[[!pageVisibility.privacy]]">
             <iron-icon icon="cr:security"></iron-icon>
--- a/chrome/browser/ui/views/profiles/profile_menu_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.cc
@@ -220,6 +220,7 @@ void ProfileMenuView::BuildMenu() {
 }
 
 gfx::ImageSkia ProfileMenuView::GetSyncIcon() const {
+  return gfx::ImageSkia();
   Profile* profile = browser()->profile();
   if (profile->IsOffTheRecord())
     return gfx::ImageSkia();
@@ -496,8 +497,7 @@ void ProfileMenuView::BuildIdentity() {
         menu_title_, menu_subtitle_);
   } else {
     menu_title_ = std::u16string();
-    menu_subtitle_ =
-        l10n_util::GetStringUTF16(IDS_PROFILES_LOCAL_PROFILE_STATE);
+    menu_subtitle_ = std::u16string();
     SetProfileIdentityInfo(
         profile_name, background_color, edit_button_params,
         ui::ImageModel::FromImage(
@@ -573,6 +573,7 @@ void ProfileMenuView::BuildSyncInfo() {
     return;
   }
 
+  return;
   // Show sync promos.
   CoreAccountInfo unconsented_account =
       identity_manager->GetPrimaryAccountInfo(signin::ConsentLevel::kSignin);
